provider/testdata/Cargo.toml

```diff
@@ -116,6 +117,7 @@ cldr_json_glob = [
     "cldr-dates-full/main/$LOCALES/timeZoneNames.json",
     "cldr-misc-full/main/$LOCALES/listPatterns.json",
     "cldr-cal-buddhist-full/main/$LOCALES/ca-buddhist.json",
+    "cldr-cal-japanese-full/main/$LOCALES/ca-japanese.json",
```

Comment:
Nit (optional): Since you seem to only be using en/ca-japanese.json right now (and not the other languages), I would add just that one file, instead of making it over all `$LOCALES`.  Optional since I know you have a follow-up to use the additional data.